test_scripts/Resumption/InteriorVehicleData/003_InteriorVD_resumption_same_moduletype_with_different_moduleIds_unexpected_disconnect.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle subscription for same moduleType with different moduleIds data
+--  after transport disconnect
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1 with modueId_1 and module_1 with modueId_2
+--
+-- Sequence:
+-- 1. Transport disconnect and reconnect are performed
+-- 2. App starts registration with actual hashId after unexpected disconnect
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(subscribe=true, module_1, moduleId_1) and
+--    RC.GetInteriorVehicleData(subscribe=true, module_1, moduleId_2) to HMI during resumption data
+-- - b. respond RAI(SUCCESS) to mobile app
```

Comment:
please add SDL get 2 successful responses for request with isSubscribed=true before SDL respond RAI(success=true, result code = SUCCESS) to mobile

Reply:
Done